### DISCUSSION TITLE
Welcome to DeepCTR Discussions!

### DISCUSSION THREAD

**Discussion by shenweichen** [shenweichen/DeepCTR#301] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Suyash-pande-89** [comment]
Hi all,
I am facing issue while creating a model with DeepFM. My code was running perfectly till 11th August 2021. But from yesterday I am getting following error while creating model:

"TypeError: Cannot convert a symbolic Keras input/output to a numpy array. This error may indicate that you're trying to pass a symbolic value to a NumPy call, which is not supported. Or, you may be trying to pass Keras symbolic inputs/outputs to a TF API that does not register dispatching, preventing Keras from automatically converting the API call to a lambda layer in the Functional Model"

![Capture](https://user-images.githubusercontent.com/65274748/129304962-400de2ef-b733-4b65-af48-12c756a6827f.PNG)

**Reply by Suyash-pande-89** [reply]
![image](https://user-images.githubusercontent.com/65274748/129305101-680729bd-7bc0-4cfa-8b3b-0385f940d10f.png)

Error seems to be in dnn_logit
